### ISSUE TITLE
Support for underscores in domain names?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#9]
```
What steps will reproduce the problem?
1.enter a domain name containing an underscore in the profile config

Error: "Windows domain can only contain alphanumeric and dash characters"

What version of the product are you using? On what operating system?

0.2.2 Ubuntu-based VirtualBox VM released 2010.08.25

Client on Windows XP SP3

Question: Is it possible to add support for underscores in domain names?
```

Original issue reported on code.google.com by `joshua.s...@gmail.com` on 2 Sep 2010 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
This is a bug and it will be fixed in the next release, thank you for finding 
it.

In the meantime, you can easily fix it yourself.  On line 178 of the 
"start-verify.html" script, add an underscore to the regex:

if( $domain !~ /^[A-Z0-9\-_]+$/i )
```

Original comment by `andrew.O...@gmail.com` on 2 Sep 2010 at 9:57
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks! That worked perfectly.
```

Original comment by `joshua.s...@gmail.com` on 2 Sep 2010 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.2.3
```

Original comment by `andrew.O...@gmail.com` on 22 Dec 2010 at 5:17
- Changed state: **Fixed**
